tools/pepsi/src/aliveness.rs

```diff
@@ -41,8 +53,32 @@ pub enum Error {
     SerializeFailed(serde_json::Error),
 }
 
-pub async fn aliveness(arguments: Arguments) -> Result<(), Error> {
-    let states = query_aliveness_list(&arguments)
+pub async fn aliveness(arguments: Arguments) -> Result<()> {
+    if arguments.watch {
+        let mut stdout = stdout();
+        stdout
+            .queue(Clear(ClearType::All))?
+            .queue(cursor::MoveTo(0, 0))?;
+        loop {
+            stdout
+                .queue(Clear(ClearType::All))?
+                .queue(cursor::MoveTo(0, 0))?
+                .queue(PrintStyledContent(
+                    format!("Aliveness (every {:?})", arguments.interval).dark_grey(),
+                ))?
+                .queue(cursor::MoveTo(0, 1))?;
+            query_and_print_aliveness(&arguments).await?;
+            tokio::time::sleep(arguments.interval).await;
```

Comment:
Replace qualified path with use?

```suggestion
            sleep(arguments.interval).await;
```